sidebarsNetwork.js

```diff
@@ -24,6 +24,17 @@ const sidebars = {
           type: "autogenerated",
           dirName: "hubble",
         },
+        {
+          type: 'category',
+          label: 'References',
```

Comment:
Can we change `References` to `Reference` (singular)? It's a bit more consistent with some of our other menus/sidebars.